airflow/providers/ssh/hooks/ssh.py

```diff
@@ -56,7 +58,9 @@ class SSHHook(BaseHook):
     :type key_file: str
     :param port: port of remote host to connect (Default is paramiko SSH_PORT)
     :type port: int
-    :param timeout: timeout for the attempt to connect to the remote_host.
+    :param conn_timeout: timeout for the attempt to connect to the remote_host.
+    :type conn_timeout: int
+    :param timeout: (Deprecated). timeout for the attempt to connect to the remote_host.
```

Comment:
Should we mention what param to use instead?

Reply:
Yeah I think it is a bit ambiguous now. with both params in the docs :)